model/config.go

```diff
@@ -18,15 +18,16 @@ import (
 const cedarConfigurationID = "cedar-system-configuration"
 
 type CedarConfig struct {
-	ID        string                    `bson:"_id" json:"id" yaml:"id"`
-	Splunk    send.SplunkConnectionInfo `bson:"splunk" json:"splunk" yaml:"splunk"`
-	Slack     SlackConfig               `bson:"slack" json:"slack" yaml:"slack"`
-	LDAP      LDAPConfig                `bson:"ldap" json:"ldap" yaml:"ldap"`
-	NaiveAuth NaiveAuthConfig           `bson:"naive_auth" json:"naive_auth" yaml:"naive_auth"`
-	CA        CAConfig                  `bson:"ca" json:"ca" yaml:"ca"`
-	Bucket    BucketConfig              `bson:"bucket" json:"bucket" yaml:"bucket"`
-	Flags     OperationalFlags          `bson:"flags" json:"flags" yaml:"flags"`
-	Service   ServiceConfig             `bson:"service" json:"service" yaml:"service"`
+	ID           string                    `bson:"_id" json:"id" yaml:"id"`
+	EvergreenURL string                    `bson:"evergreen_url" json:"evergreen_url" yaml:"evergreen_url"`
```

Comment:
I kind of want to avoid putting things like this in the top level. Perhaps we should put it in its own sub-struct so we can put auth tokens or whatever there.